apps/backend-e2e/src/admin.e2e-spec.ts

```diff
@@ -508,17 +508,194 @@ describe('Admin', () => {
 
       afterEach(() => prisma.user.deleteMany());
 
-      it('should delete a user', async () => {
+      afterAll(
+        async () =>
+          await Promise.all([
+            prisma.map.deleteMany(),
+            prisma.rank.deleteMany(),
+            prisma.run.deleteMany(),
+            prisma.report.deleteMany(),
+            prisma.activity.deleteMany()
+          ])
```

Comment:
We have a method in `DbUtil` for handling this, using a Prisma `$transaction` to let Prisma handle the deletes and avoid gridlocks (doing these all in parallel is quite likely to cause that, especially for any cascade delete relations).

Reply:
Also, `async/awaits` are unnecessary, Jest will handle any promises you return to it.